tests/integration/components/bs-form-test.js

```diff
@@ -322,6 +322,78 @@ module('Integration | Component | bs-form', function(hooks) {
     assert.dom('form .state').doesNotHaveClass('is-submitting');
   });
 
+  test('Yields #isSubmitted', async function(assert) {
+    this.actions.submit = this.stub().resolves();
```

Comment:
Wasn't sure if you prefer to use sinon to mock actions generally (#686). Could also quickly refactor to use `RSVP.resolve()` since we don't assert that the action was executed at all. IMO relying on another dependency (sinon) is just adding complexity here. But maybe you prefer it to have more similar test setups?

Reply:
That's fine.